cmd/kyverno/main.go

```diff
@@ -273,6 +274,7 @@ func main() {
 	flagset.DurationVar(&renewBefore, "renewBefore", 15*24*time.Hour, "The certificate renewal time before expiration")
 	flagset.IntVar(&maxAuditWorkers, "maxAuditWorkers", 8, "Maximum number of workers for audit policy processing")
 	flagset.IntVar(&maxAuditCapacity, "maxAuditCapacity", 1000, "Maximum capacity of the audit policy task queue")
+	flagset.IntVar(&maxAdmissionReports, "maxAdmissionReports", 10000, "Maximum number of admission reports before we stop creating new ones")
```

Comment:
Could it be confusing with the deprecated CRD admissionreports? We have ephemeralreports generated from the admission events, but no admissionreports anymore.

Reply:
admission reports are backed by the same CRD but there's a label to differentiate them from bg scan. to me the concept still exists even if it's not a separate type anymore.

Reply:
My feeling is that it can be confusing for users. I try not to use the word admissionreport when I discuss ephemeralreport issue with 1.12 users.

Reply:
My feeling is that talking about ephemeral reports is valid but sometimes too generic.

Reply:
Shall we rename it to `maxReportsForAdmission`?